exonum-java-binding/launcher-plugin/.gitignore

```diff
@@ -1,2 +0,0 @@
-# Auto-generated sources for proto messages
-exonum_java_runtime_plugin/proto/
```

Comment:
?

Reply:
This file is removed, replaced with proper one